deploy/minikube-development.sh

```diff
@@ -2,37 +2,42 @@
 
 DIR=`dirname $0`
 kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.47.3/release.yaml
-
 timeout=600 #10 minutes in seconds
 endTime=$(( $(date +%s) + timeout ))
 
-while ! oc get pods -n tekton-pipelines | grep tekton-pipelines-controller | grep Running; do
+echo -e "\033[0;32mWaiting for Tekton Pipeines to start...\033[0m"
+while ! oc get pods -n tekton-pipelines | grep tekton-pipelines-controller | grep "1/1"; do
     sleep 1
     if [ $(date +%s) -gt $endTime ]; then
         exit 1
     fi
 done
-#we need to make sure the tekton webhook has its rules installed
-while ! kubectl get mutatingwebhookconfigurations.admissionregistration.k8s.io webhook.pipeline.tekton.dev -o yaml | grep rules; do
+while ! oc get pods -n tekton-pipelines | grep tekton-pipelines-webhook | grep "1/1"; do
     sleep 1
     if [ $(date +%s) -gt $endTime ]; then
-            exit 1
+        exit 1
     fi
 done
+#we need to make sure the tekton webhook has its rules installed
+kubectl wait --for=jsonpath='{.webhooks[0].rules}' --timeout=300s mutatingwebhookconfigurations.admissionregistration.k8s.io webhook.pipeline.tekton.dev
 echo "Tekton controller is running"
 
-
-
+#CRDS are sometimes racey
+kubectl apply -k $DIR/crds/base
+#Load missing CRD
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.69.1/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+sleep 2
 
 $DIR/base-development.sh  $1
 
 # base-development.sh switches to the test-jvm-namespace namespace
-kubectl create sa pipeline
+kubectl create --dry-run=client -o=yaml sa pipeline | kubectl apply -f -
```

Comment:
Why this change?

Reply:
This is similar to prior changes I made in other scripts - this way, if the script is run twice, it is not an error condition and instead simply configuring if there are any changes, ignoring if there are not. (create versus apply).

Reply:
ah, I missed that had changed.